### PR TITLE
Added streaming support to Replicate

### PR DESCRIPTION
**Pull request by tjaffri**
Streaming support is useful if you are doing long-running completions or need interactivity e.g. for chat... adding it to replicate, using a similar pattern to other LLMs that support streaming.

Housekeeping: I ran `make format` and `make lint`, no issues reported in the files I touched.

I did update the replicate integration test but ran into some issues, specifically:

1. The original test was failing for me due to the model argument not being specified... perhaps this test is not regularly run? I fixed it by adding a call to the lightweight hello world model which should not be burdensome for replicate infra.
2. I couldn't get the `make integration_tests` command to pass... a lot of failures in other integration tests due to missing dependencies... however I did make sure the particluar test file I updated does pass, by running `poetry run pytest tests/integration_tests/llms/test_replicate.py`

Finally, I am @tjaffri https://twitter.com/tjaffri for feature announcement tweets... or if you could please tag @docugami https://twitter.com/docugami we would really appreciate that :-)

Tagging model maintainers @hwchase17  @baskaryan 

Thank for all the awesome work you folks are doing.

### CODE REVIEW

File: langchain/llms/replicate.py [hunk 1]
```diff
@@ -109,8 +112,20 @@ def _call(
             key=lambda item: item[1].get("x-order", 0),
         )
         first_input_name = input_properties[0][0]
-
         inputs = {first_input_name: prompt, **self.input}
-        iterator = replicate_python.run(self.model, input={**inputs, **kwargs})
 
-        return "".join([output for output in iterator])
+        if self.streaming:
+            prediction = replicate_python.predictions.create(
```
Comment (baskaryan) [inline langchain/llms/replicate.py:118]: why do we need to use a different method than replicate_python.run? doesn't that already stream outputs when possible (per https://replicate.com/docs/get-started/python)

**Review by baskaryan — COMMENTED** [review]
this is awesome! one comment/question

File: langchain/llms/replicate.py (hunk 1 above)
Comment (tjaffri) [reply langchain/llms/replicate.py:118]: Great catch! I'll see if I can just use the iterator returned by .run(...). Thank you!

Comment (tjaffri) [reply langchain/llms/replicate.py]: Fixed... much cleaner now, thanks.

**Comment by baskaryan** [comment]
thanks @tjaffri!

**Pull request merged**